documentation rewording and corrections

manolo opened the pull request:
Modifications based on @magi42 comments in #37

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Review on Reviewable"/>](https://reviewable.io/reviews/vaadin/vaadin-upload/42)
<!-- Reviewable:end -->

tehapo commented:
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[docs/vaadin-upload-server.adoc, line 23 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCZSr6XUJ72lFdu0aIC:-KCZSr6XUJ72lFdu0aID:1956483022) ([raw file](https://github.com/vaadin/vaadin-upload/blob/91790507de59b012fe5b555388d54387b7e368d3/docs/vaadin-upload-server.adoc#L23)):*
Looks like here has happened some copy-paste mistake or something? :smile: 

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42)*
<!-- Sent from Reviewable.io -->

manolo commented:
Review status: 2 of 3 files reviewed at latest revision, 1 unresolved discussion.

---

*[docs/vaadin-upload-server.adoc, line 23 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCZSr6XUJ72lFdu0aIC:-KCZWiebzHuU94P0lBVv:-2107409444) ([raw file](https://github.com/vaadin/vaadin-upload/blob/91790507de59b012fe5b555388d54387b7e368d3/docs/vaadin-upload-server.adoc#L23)):*
Yep, something in the clipboard seems. Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42)*
<!-- Sent from Reviewable.io -->

tehapo commented:
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[docs/vaadin-upload-server.adoc, line 23 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCZSr6XUJ72lFdu0aIC:-KC_Gr1L0lZQ89BqikI8:-1236347932) ([raw file](https://github.com/vaadin/vaadin-upload/blob/91790507de59b012fe5b555388d54387b7e368d3/docs/vaadin-upload-server.adoc#L23)):*
The "Communication" word should still be changed to start with a lowercase c.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42)*
<!-- Sent from Reviewable.io -->

manolo commented:
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[docs/vaadin-upload-server.adoc, line 23 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCZSr6XUJ72lFdu0aIC:-KC_I-weItGjpuvLvf0j:301585537) ([raw file](https://github.com/vaadin/vaadin-upload/blob/91790507de59b012fe5b555388d54387b7e368d3/docs/vaadin-upload-server.adoc#L23)):*
It was something suggested by @magi42 in #37 

`"The communication" -> "Communication" (I don't think this continues some previously started mention of communication)`

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42)*
<!-- Sent from Reviewable.io -->

tehapo commented:
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[docs/vaadin-upload-server.adoc, line 23 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCZSr6XUJ72lFdu0aIC:-KC_IglIhQqwT5_XYIsz:1595099787) ([raw file](https://github.com/vaadin/vaadin-upload/blob/91790507de59b012fe5b555388d54387b7e368d3/docs/vaadin-upload-server.adoc#L23)):*
Oh, right. But I think you should then remove "the" from the beginning? 

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42)*
<!-- Sent from Reviewable.io -->

manolo commented:
Review status: 2 of 3 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[docs/vaadin-upload-server.adoc, line 23 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCZSr6XUJ72lFdu0aIC:-KC_IwinZ7k39Fw0w2R1:-1143679403) ([raw file](https://github.com/vaadin/vaadin-upload/blob/91790507de59b012fe5b555388d54387b7e368d3/docs/vaadin-upload-server.adoc#L23)):*
Yeah, I think that is what Marko meant, Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42)*
<!-- Sent from Reviewable.io -->

tehapo commented:
Now this LGTM. Should this still be reviewed by @magi42? Or can I merge this myself?

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-:-KC_JPc1C7-lpnRvHGVA:54138323)*
<!-- Sent from Reviewable.io -->

magi42 commented:
Reviewed 2 of 3 files at r1, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 32 unresolved discussions, some commit checks failed.

---

*[docs/vaadin-upload-basic.adoc, line 44 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoeTe0DbEmOsLEZ_Di:-KCoeTe1rBr8cXrNE7s6:-1312250958) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L44)):*
* Either-or implies exclusive alternatives. Are they exclusive? (I don't know.) Doesn't "preventing default actions" change the behaviour of the process? Maybe something like: "_You can use the events to change the behaviour of the upload process, and prevent the default actions if needed._"
* "Preventing default" is maybe known to many or most JavaScript developers, but it wasn't so obvious to me, even though I remember using the method in the past. Maybe add a sentence here about it. For example: "...the upload process. _Such customizations may require preventing the default actions, which you can do with [methodname]#preventDefault()#_.

---

*[docs/vaadin-upload-basic.adoc, line 47 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoh72cDlt9iXRYpFDe:-KCoh72ddTXlTO31uMeI:1094190383) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L47)):*
* Article: "All _the_ events". Otherwise, it would mean all JavaScript events, not just vaadin-upload events.

---

*[docs/vaadin-upload-basic.adoc, line 100 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCohVtuo28LI0797hr3:-KCohVtuo28LI0797hr4:-170394228) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L100)):*
* I think "would" -> "will"

---

*[docs/vaadin-upload-basic.adoc, line 101 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCohlRZ37vz16rA329B:-KCohlRZ37vz16rA329C:1714240602) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L101)):*
* "would" -> "will" also here

---

*[docs/vaadin-upload-basic.adoc, line 102 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCohptiAkyPIXt_DV2w:-KCohptiAkyPIXt_DV2x:-43275157) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L102)):*
* No emphasis for XHR

---

*[docs/vaadin-upload-basic.adoc, line 106 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoi-s1lyFmVYf7azEt:-KCoi-s1lyFmVYf7azEu:1693664745) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L106)):*
* Again odd word order. Rather: "_If the default is prevented_" or rather use active voice: "_If you prevent the default action_"
* Usually there's no need for "then" after the comma; it is usually better left as implicit. (It can be used to emphasize the conditional nature of the sentence and the result of the condifion, but maybe this doesn't need that.)

---

*[docs/vaadin-upload-basic.adoc, line 108 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KComOTFlDQ5OCECHn42:-KComOTFlDQ5OCECHn43:850946104) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L108)):*
* Again odd word order. Rather: "_If the default is prevented_" or rather use active voice: "_If you prevent the default action_"

---

*[docs/vaadin-upload-basic.adoc, line 111 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoPkhBe3er_QITSA2T:-KCoPkhCTuanJqoqmmSM:-19350534) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L111)):*
The event names in this list should be emphasized somehow. There's no style name for them, so just use the literal style with `` `backticks` ``.

---

*[docs/vaadin-upload-basic.adoc, line 111 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KComdSZmflC0Vs5QfJF:-KComdSZmflC0Vs5QfJG:-1908473564) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L111)):*
* "success" -> "succeeds"

---

*[docs/vaadin-upload-custom.adoc, line 11 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KComqeUOGvGjaggtHt4:-KComqeUOGvGjaggtHt5:1919699890) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L11)):*
* "changing" -> "by changing"
* "providing" -> "by providing"
* No comma in a list of two items (remove it before "or")

---

*[docs/vaadin-upload-custom.adoc, line 37 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConkj8jL6JMGN-5JDh:-KConkj8jL6JMGN-5JDi:330540076) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L37)):*
* Capitalize "_Properties_"

---

*[docs/vaadin-upload-custom.adoc, line 51 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConpABNtUN5FOtOwho:-KConpABNtUN5FOtOwhp:330540076) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L51)):*
* Capitalize "_Properties_"

---

*[docs/vaadin-upload-custom.adoc, line 55 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConqedXOHT-Ymzd-i0:-KConqedXOHT-Ymzd-i1:111032137) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L55)):*
* Capitalize "_Element_"

---

*[docs/vaadin-upload-custom.adoc, line 64 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConwCkfK3fRZhphdIf:-KConwCpDNXzQ7gIhsbI:2142013321) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L64)):*
* What is "overing"? Covering? Hovering? Lowering?

---

*[docs/vaadin-upload-custom.adoc, line 70 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConeKIURMJqcaineQy:-KConeKMGymnrgpL1E7w:111032137) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L70)):*
* Capitalize "_Element_"

---

*[docs/vaadin-upload-custom.adoc, line 89 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConYrMFre2ea6qeNQv:-KConYrMFre2ea6qeNQw:-1329643918) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L89)):*
* Capitalize "Content"

---

*[docs/vaadin-upload-custom.adoc, line 91 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCooBWFRM8xrfAKnX63:-KCooBWFRM8xrfAKnX64:1110098880) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L91)):*
* "_The [elementname]#vaadin-upload# element_"

---

*[docs/vaadin-upload-custom.adoc, line 93 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCooZISkH9pQSs5RYZD:-KCooZISkH9pQSs5RYZE:-1316349168) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L93)):*
Rather complex sentence, I have no clue what it means... I don't understand if a) the content, which has a drop label, is before the file list, b) the content, which has a file list and a drop label before it, or c) the drop label before the file list and the content part after it. (I don't know how a drop label is "with" the content)

---

*[docs/vaadin-upload-custom.adoc, line 94 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCorp-SLHXnAe_kjkk3:-KCorp-WB8CIc2yWVGUO:-83603186) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L94)):*
* "composed with" is an odd expression. Should that be "composed of"?

---

*[docs/vaadin-upload-custom.adoc, line 95 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCorb1AlZQ5InF8RazT:-KCorb1AlZQ5InF8RazU:1532527385) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L95)):*
* Subject of the clause is missing: "but _it_ will be"

---

*[docs/vaadin-upload-server.adoc, line 13 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoslokpm3zYU3qA6nd:-KCoslol7xYllwbLKXvN:1434254369) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L13)):*
If XMLHttpRequest is not a unique singleton, but an instance, use indefinite article ("an").

---

*[docs/vaadin-upload-server.adoc, line 17 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCosXlu7br8gZh3yM2C:-KCosXlu7br8gZh3yM2D:-1192628949) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L17)):*
XHR is not really a classname here, but rather an acronym, so maybe no emphasis for it.

---

*[docs/vaadin-upload-server.adoc, line 19 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCot5L5pvUy2_LwQYeR:-KCot5L5pvUy2_LwQYeS:-1989180854) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L19)):*
* Comma missing after introductory clause: "...-side, we use"

---

*[docs/vaadin-upload-server.adoc, line 23 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCotNoEqglinegpy7gG:-KCotNoEqglinegpy7gH:1843056686) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L23)):*
* I think there's articles missing from these: "_the [elementname]#vaadin-upload# element and the server-side_"

---

*[docs/vaadin-upload-server.adoc, line 47 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoFHbsZFcD9PoqhmQB:-KCoFHbt4vO_LQtGO_OH:-2110173202) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L47)):*
* Capitalize "Request"
* "via" is maybe a bit odd word, rather use "through" or "with"

---

*[docs/vaadin-upload-server.adoc, line 49 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCo9qtMZGrEeMt84r6f:-KCo9qtNhsKYuygAUBE8:687744952) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L49)):*
* The acronym XHR should not be emphasized here.
* Is the "response" some sort of ID here? If not, do not emphasize it, but talk about the response as a concept.

Maybe rather: "There are a number of events that allow modifying the XHR before sending it, or the response received from the request."

---

*[docs/vaadin-upload-server.adoc, line 51 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoAXn4E5fB9m5XakhK:-KCoAXn5tpQ8hwLNiisv:728106165) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L51)):*
* Emphasis missing from `upload-before`. Just use literal (backquotes) emphasis if there's nothing more applicable.
* Comma after introductory clause: "_At this point, _"
* "...URL changing..." -> "_...URL by setting the [propertyname]#file.uploadTarget# attribute._"
* Article: "prevent _the_ default action."
* "Could" is oddly speculatory. Maybe it's more clear to tell how rather than to use a conditional. So: "_You can also reject the file at this point by preventing the default action._"

---

*[docs/vaadin-upload-server.adoc, line 67 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoGyoq_026MJCUfJ6c:-KCoGyoq_026MJCUfJ6d:-250095743) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L67)):*
* Emphasize `upload-request`.
* Maybe rather as previously: "_You can cancel sending the request to the server by preventing the default action._"

---

*[docs/vaadin-upload-server.adoc, line 77 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCovAefBFhWNDhEylKh:-KCovAegQsEOweSAtOli:928173834) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L77)):*
* Emphasize `upload-response`
* "when" refers to the time when the server message was received, but "using" an event is usually done when you add the listener. So rather: "_The event is dispatched when the server message was received, but before the component processes it._"

---

*[docs/vaadin-upload-server.adoc, line 78 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCowBkUj1BZjBlMaCY0:-KCowBkUj1BZjBlMaCY1:-529181005) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L78)):*
* -> "make the upload fail _by_ setting"

---

*[docs/vaadin-upload-server.adoc, line 79 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCowUyl9CNYXsFq6Qrk:-KCowUymYifnonwAlgsu:-767099406) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L79)):*
* "Also you can" means that you can do it like other people can. I think you mean: "_Also, you can..._"
* Maybe rather rephrase: "_If you prevent the default action, the response is not checked for errors and the UI is not updated._"

---

*[docs/vaadin-upload-server.adoc, line 94 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoxQUvN74N9eAk_lb0:-KCoxQUwRhS27qbPI9g-:522829220) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L94)):*
* Comma: "_Nevertheless, we..._"
* Leaving links maybe sounds like a nice playful expression... The word "here" is always problematic in text. You could also say: "_we leave you with some links_"

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42)*
<!-- Sent from Reviewable.io -->

manolo commented:
Review status: 0 of 3 files reviewed at latest revision, 32 unresolved discussions, some commit checks failed.

---

*[docs/vaadin-upload-basic.adoc, line 44 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoeTe0DbEmOsLEZ_Di:-KCyHoFhcCM6H3E6s4Lj:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L44)):*
Done.

---

*[docs/vaadin-upload-basic.adoc, line 47 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoh72cDlt9iXRYpFDe:-KCyHqWDXn0ZWPSFQuXA:-682492604) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L47)):*
Done.  But why do we need the with plurals, I learnt that it is not needed unless there is specificity like 'all the big events' 

---

*[docs/vaadin-upload-basic.adoc, line 100 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCohVtuo28LI0797hr3:-KCzYVNs-kVerimaAKfB:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L100)):*
Done.

---

*[docs/vaadin-upload-basic.adoc, line 101 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCohlRZ37vz16rA329B:-KCzYXymhKaj22zkeOvi:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L101)):*
Done.

---

*[docs/vaadin-upload-basic.adoc, line 102 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCohptiAkyPIXt_DV2w:-KCzYbKP8QWxe1KFHJwD:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L102)):*
Done.

---

*[docs/vaadin-upload-basic.adoc, line 106 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoi-s1lyFmVYf7azEt:-KCzZcXgXzUHvYR1Eez1:804027706) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L106)):*
I insist, for a JS developer is the event what is 'defaultPrevented' so it makes perfect sense, setting defaultPrevented flag to true does not necessary mean that the default behaviour could be prevented, it is a matter of the component checking that flag.
I have took a look to other documentation, and seems that it's more correct to use default-prevented as a entire word, so replacing 'default prevented' by default-prevented, which is more meaningful for a JS developer ie: https://developer.chrome.com/apps/tags/webview

---

*[docs/vaadin-upload-basic.adoc, line 108 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KComOTFlDQ5OCECHn42:-KCza5sfbQ5x2FaILzIg:-564674093) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L108)):*
changed by default-prevented

---

*[docs/vaadin-upload-basic.adoc, line 111 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoPkhBe3er_QITSA2T:-KCzbA0XpqOfi0RAXA6J:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L111)):*
Done.

---

*[docs/vaadin-upload-basic.adoc, line 111 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KComdSZmflC0Vs5QfJF:-KCzbEmQ2Vbgvw2Z7Uos:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-basic.adoc#L111)):*
Done.

---

*[docs/vaadin-upload-custom.adoc, line 11 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KComqeUOGvGjaggtHt4:-KCzbPUcciuBKBVRbTlx:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L11)):*
Done.

---

*[docs/vaadin-upload-custom.adoc, line 37 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConkj8jL6JMGN-5JDh:-KCzb_1kTLyWlm0JEOi9:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L37)):*
Done.

---

*[docs/vaadin-upload-custom.adoc, line 51 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConpABNtUN5FOtOwho:-KCzb_Lab96GH5GVIKMq:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L51)):*
Done.

---

*[docs/vaadin-upload-custom.adoc, line 55 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConqedXOHT-Ymzd-i0:-KCzb_iMg4jfur5wNiWw:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L55)):*
Done.

---

*[docs/vaadin-upload-custom.adoc, line 64 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConwCkfK3fRZhphdIf:-KCzcj2iAXurGd2XjxQN:468505677) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L64)):*
hovering, Done.

---

*[docs/vaadin-upload-custom.adoc, line 70 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConeKIURMJqcaineQy:-KCzcnI16n09qfBjpTFr:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L70)):*
Done.

---

*[docs/vaadin-upload-custom.adoc, line 89 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KConYrMFre2ea6qeNQv:-KCzdandSkrbWrabBF2c:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L89)):*
Done.

---

*[docs/vaadin-upload-custom.adoc, line 91 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCooBWFRM8xrfAKnX63:-KCzdfQCFvUf3BUKJiQy:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L91)):*
Done.

---

*[docs/vaadin-upload-custom.adoc, line 93 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCooZISkH9pQSs5RYZD:-KCzgev1402sRZ48lZ0E:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L93)):*
Done.

---

*[docs/vaadin-upload-custom.adoc, line 94 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCorp-SLHXnAe_kjkk3:-KCzgfR3ADYhQt3KWoMZ:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L94)):*
Done.

---

*[docs/vaadin-upload-custom.adoc, line 95 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCorb1AlZQ5InF8RazT:-KCzgk-ep9NAmcsIrH5R:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L95)):*
Done.

---

*[docs/vaadin-upload-server.adoc, line 13 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoslokpm3zYU3qA6nd:-KCzssj8bWvrneeV5rMu:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L13)):*
Done.

---

*[docs/vaadin-upload-server.adoc, line 17 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCosXlu7br8gZh3yM2C:-KCzhSghfJeJ4QQVhFb8:-1920798693) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L17)):*
it should be XMLHttpRequest, because we are talking about classes here. Changed

---

*[docs/vaadin-upload-server.adoc, line 19 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCot5L5pvUy2_LwQYeR:-KCzhhdETMn3PsK_wJLa:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L19)):*
Done.

---

*[docs/vaadin-upload-server.adoc, line 23 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCotNoEqglinegpy7gG:-KCzhmQyreX5O3ujjmOA:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L23)):*
Done.

---

*[docs/vaadin-upload-server.adoc, line 47 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoFHbsZFcD9PoqhmQB:-KCzhtVcV_Pe0Uhd4L6K:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L47)):*
Done.

---

*[docs/vaadin-upload-server.adoc, line 49 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCo9qtMZGrEeMt84r6f:-KCzhxJHFsGEO-ePlkNB:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L49)):*
Done.

---

*[docs/vaadin-upload-server.adoc, line 51 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoAXn4E5fB9m5XakhK:-KCzqc19cOFEoSTINyfa:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L51)):*
Done.

---

*[docs/vaadin-upload-server.adoc, line 67 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoGyoq_026MJCUfJ6c:-KCzratnL5XADdibW5vZ:-1155501846) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L67)):*
That changes the meaning, rewording a little bit the sentence.

---

*[docs/vaadin-upload-server.adoc, line 77 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCovAefBFhWNDhEylKh:-KCzs6IrSBqP2zHFDOro:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L77)):*
Done.

---

*[docs/vaadin-upload-server.adoc, line 78 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCowBkUj1BZjBlMaCY0:-KCzswZsuF1S3-9QLY5I:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L78)):*
Done.

---

*[docs/vaadin-upload-server.adoc, line 79 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCowUyl9CNYXsFq6Qrk:-KCzsTk4PCUqIs0nTi4z:1838771892) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L79)):*
using default-prevented here, makes more sense. Done in this way

---

*[docs/vaadin-upload-server.adoc, line 94 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCoxQUvN74N9eAk_lb0:-KCzsjSsrh8sjWoLPODd:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L94)):*
Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42)*
<!-- Sent from Reviewable.io -->

magi42 commented:
Reviewed 3 of 3 files at r4.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[docs/vaadin-upload-custom.adoc, line 91 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCooBWFRM8xrfAKnX63:-KCzz37bZOWs61Ie1rUk:1177262610) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L91)):*
The word "element" seems to have disappeared from there. 

---

*[docs/vaadin-upload-server.adoc, line 79 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCowUyl9CNYXsFq6Qrk:-KD--BpoimPhEipIw2f1:103009681) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L79)):*
Comma missing after introductory condition: "_default-prevented, the_" (it's an if-then expression, but the "then" should usually be omitted.)

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42)*
<!-- Sent from Reviewable.io -->

manolo commented:
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[docs/vaadin-upload-custom.adoc, line 91 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCooBWFRM8xrfAKnX63:-KD-1WJUcfv9MkIV0mFa:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-custom.adoc#L91)):*
Done.

---

*[docs/vaadin-upload-server.adoc, line 79 \[r3\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-KCowUyl9CNYXsFq6Qrk:-KD-1chHPK1yMMcIHPlq:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/a02070639e86c3d22d116dda8852a96f12eb5e5f/docs/vaadin-upload-server.adoc#L79)):*
Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42)*
<!-- Sent from Reviewable.io -->

magi42 commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 2 of 2 files at r5.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42#-:-KD-26qFBZ2g5A1f8i5P:1417088721)*
<!-- Sent from Reviewable.io -->

manolo commented:
Reviewed 1 of 3 files at r4, 1 of 2 files at r5, 1 of 1 files at r6.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/42)*
<!-- Sent from Reviewable.io -->
